GKI variables and serial plot labels

jeremiet-1 opened the pull request:
Hello, this is work I've done to extend the code for GKI variables and add bin labels to some of the serial plots that didn't have them before.

Commit:
GKI variables and serial plot labels

epelaaez commented on GeneratorAnalysis/Scripts/.#Constants.h:
Can you remove this file? (also `.#script_LoopGenerators.C`)

epelaaez commented on GeneratorAnalysis/Scripts/Constants.h:
```diff
@@ -56,6 +56,35 @@ namespace Constants {
 
+    // Variables for double differential analysis - GKI case
+    static const int TwoDNBinsMuonCosThetaThreeD = 2; 
+    std::vector<double> TwoDArrayNBinsMuonCosThetaThreeD{-1.0,0.5,1.0};
```
Is there a difference between these variables and `TwoDArrayNBinsMuonCosTheta`/`TwoDNBinsMuonCosTheta` defined above?

epelaaez commented on GeneratorAnalysis/Selections/TwoPTools.cxx:
```diff
@@ -31,16 +31,37 @@ TwoPTools::TwoPTools(TVector3 MuonVector, TVector3 LeadingProtonVector, TVector3
+    // GKI vectors
+    double TotalProtonMomentum = TotalProtonVector.Mag();
+    double Ecal = (TMath::Sqrt((MuonVector.Mag()*MuonVector.Mag()) + (0.106*0.106)) + 
+        TMath::Sqrt((TotalProtonMomentum*TotalProtonMomentum) + (0.938272*0.938272)) - 0.938272 + 0.0309);
```
Maybe we should add a comment saying what the constants here represent?

epelaaez reviewed (commented):
Looks pretty good! I added a few comments/questions. Also, in some recent commits I moved the `GeneratorAnalysis/Scripts/Constants.h` file into `Utils/Constants.h`, so you should probably pull that change and make sure nothing breaks. And I'll also make a change so that we don't have to explicitly write `epelaez`/`theobal1` everytime we need it, but rather make it a constants in `Constants.h` to make life easier.

epelaaez commented:
In a4eacf74202163d15e038eeef9296de8ef2dfeb4, I moved the username declaration to the constants file, that way you only have to modify the user once instead of every time we want to access/save files.

Commit:
Merge branch 'main' of github.com:epelaaez/CC1muAnalysis

Commit:
Edits from pull request

Commit:
Merging, included Constants.h to GeneratorOverlay file

Commit:
Merge branch 'main' of github.com:epelaaez/CC1muAnalysis

epelaaez commented on GeneratorAnalysis/Scripts/FlatTreeAnalyzer.h:
```diff
@@ -5,6 +5,7 @@
 #include <TChain.h>
 #include <TFile.h>
 #include <TString.h>
+#include "../../Utils/Constants.h"
```
I think this got duplicated during some merging, sorry about that

epelaaez commented on GeneratorAnalysis/Scripts/GeneratorInteBreakDown.cpp:
```diff
@@ -46,8 +46,8 @@ void GeneratorInteBreakDown() {
 
-    Names.push_back(OutFilePath+"FlatTreeAnalyzerOutput_GENIE_AR23.root"); 
-    Labels.push_back("GENIE_AR23");
+    /*Names.push_back(OutFilePath+"FlatTreeAnalyzerOutput_GENIE_AR23.root"); 
+    Labels.push_back("GENIE_AR23");*/
```
One last thing, can we make it so that the original list of names/labels for the generators remains the same?

Commit:
Edits for merging

Commit:
Merge branch 'main' of github.com:epelaaez/CC1muAnalysis

Commit:
Merge branch 'main' of github.com:epelaaez/CC1muAnalysis

Commit:
Merge branch 'main' of github.com:epelaaez/CC1muAnalysis

Commit:
GKI variables for CAFAnaAnalysis

Commit:
Merge branch 'main' of github.com:epelaaez/CC1muAnalysis

Commit:
plots before fix

Commit:
Minor fixes, new plots

Conflicts:
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrMuonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrMuonMomentum.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CorrTransverseMomentum.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovEventCount.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovMuonMomentum.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/CovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/FracCovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/AhtBY_multisigma_Genie/FracCovSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CorrCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CorrCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CorrDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CorrLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CorrLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CorrMuonCosTheta.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CorrMuonMomentum.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CorrRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CorrRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CorrTransverseMomentum.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CovEventCount.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CovMuonMomentum.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/CovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/FracCovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/FracCovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/FracCovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/FracCovEventCount.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/FracCovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/FracCovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/FracCovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/FracCovMuonMomentum.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/FracCovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/FracCovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/BhtBY_multisigma_Genie/FracCovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CorrCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CorrCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CorrDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CorrLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CorrLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CorrMuonCosTheta.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CorrMuonMomentum.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CorrRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CorrRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CorrTransverseMomentum.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CovEventCount.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CovMuonMomentum.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/CovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/FracCovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/FracCovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/FracCovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/FracCovEventCount.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/FracCovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/FracCovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/FracCovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/FracCovMuonMomentum.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/FracCovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/FracCovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/CV1uBY_multisigma_Genie/FracCovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CorrCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CorrCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CorrDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CorrLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CorrMuonCosTheta.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CorrRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CorrTransverseMomentum.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CovEventCount.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CovMuonMomentum.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/CovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/FracCovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/FracCovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/FracCovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/FracCovEventCount.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/FracCovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/FracCovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/FracCovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/CV2uBY_multisigma_Genie/FracCovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/Detector/CovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/Detector/CovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/Detector/CovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/Detector/CovEventCount.png
	Figs/CAFAna/Uncertainties/Detector/CovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/CovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/Detector/CovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/CovMuonMomentum.png
	Figs/CAFAna/Uncertainties/Detector/CovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/CovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/Detector/CovSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/CovSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/CovSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/CovSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/CovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/Detector/FracCovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/Detector/FracCovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/Detector/FracCovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/Detector/FracCovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/FracCovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/Detector/FracCovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/FracCovMuonMomentum.png
	Figs/CAFAna/Uncertainties/Detector/FracCovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/FracCovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/Detector/FracCovSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/FracCovSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/FracCovSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/FracCovSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Detector/FracCovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/MCStat/CorrCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/MCStat/CorrLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CorrMuonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CorrRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CorrSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CorrSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CorrSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CorrSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/MCStat/CovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/MCStat/CovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/MCStat/CovEventCount.png
	Figs/CAFAna/Uncertainties/MCStat/CovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/MCStat/CovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CovMuonMomentum.png
	Figs/CAFAna/Uncertainties/MCStat/CovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/MCStat/CovSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CovSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CovSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CovSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/MCStat/CovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/NTargets/CovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/NTargets/CovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/NTargets/CovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/NTargets/CovEventCount.png
	Figs/CAFAna/Uncertainties/NTargets/CovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/CovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/NTargets/CovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/CovMuonMomentum.png
	Figs/CAFAna/Uncertainties/NTargets/CovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/CovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/NTargets/CovSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/CovSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/CovSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/CovSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/CovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovMuonMomentum.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/NTargets/FracCovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/POT/CovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/POT/CovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/POT/CovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/POT/CovEventCount.png
	Figs/CAFAna/Uncertainties/POT/CovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/CovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/POT/CovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/CovMuonMomentum.png
	Figs/CAFAna/Uncertainties/POT/CovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/CovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/POT/CovSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/CovSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/CovSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/CovSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/CovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/POT/FracCovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/POT/FracCovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/POT/FracCovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/POT/FracCovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/FracCovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/POT/FracCovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/FracCovMuonMomentum.png
	Figs/CAFAna/Uncertainties/POT/FracCovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/FracCovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/POT/FracCovSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/FracCovSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/FracCovSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/FracCovSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/POT/FracCovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovEventCount.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovMuonMomentum.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/CovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovMuonMomentum.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Reinteraction/FracCovTransverseMomentum.png
	Figs/CAFAna/Uncertainties/Statistical/CovCosOpeningAngleMuonTotalProton.png
	Figs/CAFAna/Uncertainties/Statistical/CovCosOpeningAngleProtons.png
	Figs/CAFAna/Uncertainties/Statistical/CovDeltaAlphaT.png
	Figs/CAFAna/Uncertainties/Statistical/CovEventCount.png
	Figs/CAFAna/Uncertainties/Statistical/CovLeadingProtonCosTheta.png
	Figs/CAFAna/Uncertainties/Statistical/CovLeadingProtonMomentum.png
	Figs/CAFAna/Uncertainties/Statistical/CovMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Statistical/CovMuonMomentum.png
	Figs/CAFAna/Uncertainties/Statistical/CovRecoilProtonCosTheta.png
	Figs/CAFAna/Uncertainties/Statistical/CovRecoilProtonMomentum.png
	Figs/CAFAna/Uncertainties/Statistical/CovSerialCosOpeningAngleMuonTotalProton_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Statistical/CovSerialCosOpeningAngleProtons_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Statistical/CovSerialDeltaAlphaT_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Statistical/CovSerialTransverseMomentum_InMuonCosTheta.png
	Figs/CAFAna/Uncertainties/Statistical/CovTransverseMomentum.png
	Utils/Constants.h

Commit:
Merge branch 'main' of github.com:epelaaez/CC1muAnalysis, minor resolution

jeremiet-1 committed:
Delete v09_75_03 directory

Commit:
Merge branch 'main' of github.com:epelaaez/CC1muAnalysis

Commit:
Merge branch 'main' of github.com:jeremiet-1/CC1muAnalysis

Commit:
Merge branch 'main' of github.com:jeremiet-1/CC1muAnalysis

epelaaez commented on CAFAnaAnalysis/MCC9_FluxHist_volTPCActive.root:
This file is only needed under `Utils/` now

Commit:
Delete extra source file

epelaaez commented on GeneratorAnalysis/Scripts/FlatTreeAnalyzer.cxx:
```diff
@@ -46,7 +46,8 @@ void FlatTreeAnalyzer::Loop() {
 
     // Output file
-    TString Directory = "/pnfs/sbnd/persistent/users/" + (TString)UserName + "/HighSamples/";
+    TString Directory = "/pnfs/sbnd/persistent/users/theobal1/HighSamples/";
+    //TString Directory = "/pnfs/sbnd/persistent/users/" + (TString)UserName + "/HighSamples/";
```
I think for merging, we want to keep the `+(TString)UserName+` part to make it easy for other people to use

epelaaez commented:
I think assuming this is up to date with main and has no merge conflicts, it looks pretty much good to be merged in!

Commit:
Tstring path fix

Commit:
MEC Overlay variables
